agent/agent_test.go

```diff
@@ -546,8 +546,8 @@ func TestRunnerRun(t *testing.T) {
 			Recipe:      validRecipe,
 			RecordCount: 1,
 			Success:     true,
-			// DurationInMs is 0 because of the process took below a millisecond, in production scenario it takes longer than that
-			DurationInMs: 0,
+			// DurationInMs is 1000 as an arbitary alloted duration time, in production scenario it takes longer than that
```

Comment:
we can also remove this comment since the variable name should be pretty descriptive